.changeset/chatty-sloths-join.md

```diff
@@ -0,0 +1,7 @@
+---
+"@cypress-design/icon-registry": patch
+"@cypress-design/icon-vue": patch
+"@cypress-design/icon-react": patch
```

Comment:
```suggestion
"@cypress-design/vue-icon": patch
"@cypress-design/react-icon": patch
```

you could also have used the comand `yarn changesset` to avoid having to look for package names